main.go

```diff
@@ -23,6 +25,10 @@ var (
 	source      = "https://github.com/giantswarm/credentiald"
 )
 
+func init() {
+	rand.Seed(time.Now().UnixNano())
+}
```

Comment:
Is this how it should be? I remember there was some weird best practise but I do not recall. Like multiple packages can do this and set the seed to something static for testing and then you are fucked so the whole random magic should be injectable and configured where it is used and not defined on a global package level. IMO this will come back in really bad ways eventually.

Reply:
Uncertain. I couldn't find anything via Google.